### ISSUE TITLE
When the first test in a file fails, it is shown as a failure on the previous line

### ISSUE THREAD

**Issue by kbond** [mikeerickson/phpunit-pretty-result-printer#8]
I have two tests:

```php
final class FirstFileTest extends \PHPUnit\Framework\TestCase
{
    public function test_first_test()
    {
        $this->assertTrue(true);
    }

    public function test_second_test()
    {
        $this->assertTrue(true);
    }
}
```

And:

```php
final class SecondFileTest extends \PHPUnit\Framework\TestCase
{
    public function test_third_test()
    {
        $this->assertTrue(false); // failure
    }

    public function test_fourth_test()
    {
        $this->assertTrue(true);
    }
}
```

The output is as follows:

```
 ==> FirstFileTest                               ✓ ✓ ✖ 
 ==> SecondFileTest                              ✓ 
```

**Comment by kbond** [comment]
When I make `test_first_test` fail, the output is as follows:

```
✖ 
 ==> FirstFileTest                               ✓ ✖ 
 ==> SecondFileTest                              ✓
```

**Comment by mikeerickson** [comment]
@kbond I have been able to reproduce your issue, I will try and get this worked out over the weekend, but we are starting our holiday break today so not 100% sure

**Comment by kbond** [comment]
Great, no rush!

**Comment by mikeerickson** [comment]
If anybody would like to help this issue, it would be most welcome.  If not, I plan on getting to this ni the next few days.

**Comment by mikeerickson** [comment]
Released `0.4.4`
